readthedocs/api/v2/views/footer_views.py

```diff
@@ -25,7 +25,7 @@ def get_version_compare_data(project, base_version=None):
     :param base_version: We assert whether or not the base_version is also the
                          highest version in the resulting "is_highest" value.
     """
-    versions_qs = Version.objects.public(project=project)
+    versions_qs = Version.internal.public(project=project)
```

Comment:
The main blocker is currently that `public()` is removing the filtering put in place by `internal` (https://github.com/rtfd/readthedocs.org/pull/5750#issuecomment-503210870). We need to figure out that fix before this can be merged.